MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -1476,7 +1476,7 @@
     _properties.limits.maxComputeWorkGroupCount[1] = kMVKUndefinedLargeUInt32;
     _properties.limits.maxComputeWorkGroupCount[2] = kMVKUndefinedLargeUInt32;
 
-    _properties.limits.maxDrawIndexedIndexValue = numeric_limits<uint32_t>::max();
+    _properties.limits.maxDrawIndexedIndexValue = kMVKUndefinedLargeUInt32;
```

Comment:
Why? If the `fullDrawIndexUint32` is supported, this is required to be the maximum `uint32_t` value. That's why I changed it to set it to `numeric_limits<uint32_t>::max()` in the first place. DXVK requires this for feature level 10 support, because all D3D10+ hardware supports it, and I'm pretty sure Metal (at least, on desktop) requires D3D11+ hardware.

Reply:
Shoot! My bad. I just interpreted it as a typo...without fully absorbing how that limit is special.

Fixing it now...

Reply:
Thanks for catching that! 🤦🏻‍♂️😏